src/tests/helper.rs

```diff
@@ -29,6 +32,17 @@ use crate::{
     Sha256Jmt, ValueHash, SPARSE_MERKLE_PLACEHOLDER_HASH,
 };
 
+type MockUpdateMerkleProof = UpdateMerkleProof<
+    CoreWrapper<
+        CtVariableCoreWrapper<
+            Sha256VarCore,
+            UInt<UInt<UInt<UInt<UInt<UInt<UTerm, B1>, B0>, B0>, B0>, B0>, B0>,
+            OidSha256,
+        >,
+    >,
```

Comment:
nit: I think we can use the `type` alias that the sha256 crate exposes: `type MockUpdateMerkleProof = UpdateMerkleProof<Sha256, Vec<u8>>`. Am I missing something? 

Also, this doesn't look like a Mock to me - it seems like a real proof. Maybe we should rename?